source/app/service-providers/commands/export.ts

```diff
@@ -74,7 +75,23 @@ export default class Export extends ZettlrCommand {
       if (fileDescriptor !== null) {
         exporterOptions.sourceFiles.push(fileDescriptor)
         exporterOptions.cwd = fileDescriptor.dir
-        exporterOptions.targetDirectory = (exportTo === 'temp') ? app.getPath('temp') : fileDescriptor.dir
+        switch (exportTo) {
+          case 'ask': {
+            const focusedWindow = BrowserWindow.getFocusedWindow()
+            const folderSelection = await this._app.windows.askDir(trans('system.export_dialog.title'), focusedWindow, trans('system.export_dialog.save'))
```

Comment:
You're only dealing with exports, right? In that case I think you can ditch the focusedWindow stuff, reduce the imports here by one module, and keep it leaner. Otherwise, we can also think about maybe changing the `askDir` function itself